cmd/plugin_test.go

```diff
@@ -443,6 +451,93 @@ func TestPluginDirectRead(t *testing.T) {
 	}
 }
 
+// We ran into a bug where the start time for the transfer was not recorded correctly and was almost always the same as the end time
+// (since they were set at similar sections of code). This test ensures that they are different and that the start time is before the end time.
+func TestPluginCorrectStartAndEndTime(t *testing.T) {
+	viper.Reset()
+	viper.Set("ConfigDir", t.TempDir())
+	server_utils.ResetOriginExports()
+	defer viper.Reset()
+	defer server_utils.ResetOriginExports()
+	err := config.InitClient()
+	require.NoError(t, err)
```

Comment:
Brian recently added `test_utils.InitClient` that handles most of this preamble. Can we modify the test to use that?